docs/mindsdb-docs/docs/contribute/integrations.md

```diff
@@ -77,7 +76,7 @@ Whenever you want to parse a string that contains SQL, we strongly recommend usi
 
 Most handlers need to store internal metadata ranging from a list of registered tables to implementation-specific details that greatly vary from one case to another.
 
-The recommendation for storing these bits of information is to use storage handlers from the `integrations.libs.storage_handler` library. We currently support two options: an SQLite backend and a Redis backend. In both cases, the premise is the same: a key-value store system is set up such that interfaces are simple and clean, exposing only the `get()` and `set()` methods for usage within the data and predictive handlers.
+The recommendation for storing these bits of information is to use storage handlers from the `integrations.libs.storage_handler` library. We currently support two options: an SQLite backend and a Redis backend. In both cases, the premise is the same: a key-value store system is set up such that interfaces are simple and clean, exposing only the `get()` and `set()` methods for usage within the handlers.
```

Comment:
This is outdated 😅. We now offer a very different mechanism to store internal state, and AFAIK only for ML handlers, as DB handlers are stateless. @ea-rus @StpMax would you say the following is an accurate replacement for the previous paragraph and this one:

> ML handlers may need to store internal metadata. For this, you should leverage `interfaces.storage.model_fs`, which exposes a simple interface (e.g. `get()` and `set()` methods) for usage within ML handlers.

Reply:
agree, need to remove any mention of `storage_handler` here